### DISCUSSION TITLE
Is there any way to be notified via mail when ever there is crash in app?

### DISCUSSION THREAD

**Discussion by Yandamuri** [invertase/react-native-firebase#6609] [Q&A] [answered]
I have configured `@react-native-firebase/crashlytics` in my app. Able to see the crash reports inside `Firebase Console`. However, It would be so flexible if crashes are notified via mail.

Hence, Please let me know if there is such a setting.

**Comment by mikehardy** [comment, marked as answer]
Yep - they are called crashlytics alerts

https://search.brave.com/search?q=crashlytics+alerts

https://firebase.google.com/docs/crashlytics/velocity-alerts

**Reply by Yandamuri** [reply]
Great. Thank you @mikehardy
